ArchiSteamFarm/Bot.cs

```diff
@@ -1177,10 +1177,18 @@ public sealed class Bot : IDisposable {
 
 			while (i < message.Length) {
 				int partLength;
+				bool copyNewline = false;
 
 				if (message.Length - i > maxMessageLength) {
 					int lastNewLine = message.LastIndexOf(Environment.NewLine, i + maxMessageLength, maxMessageLength, StringComparison.Ordinal);
-					partLength = lastNewLine > i ? lastNewLine - i : maxMessageLength;
+
+					if (lastNewLine > i) {
+						partLength = lastNewLine - i + Environment.NewLine.Length;
```

Comment:
You'll run into 5001 chars here if your newline character will be at 5000 position.